src/main/java/com/example/bankservicedemo/controller/AccountController.java

```diff
@@ -0,0 +1,41 @@
+package com.example.bankservicedemo.controller;
+
+import com.example.bankservicedemo.mapper.AccountMapper;
+import com.example.bankservicedemo.model.dto.account.AccountRequestDto;
+import com.example.bankservicedemo.model.dto.account.AccountResponseDto;
+import com.example.bankservicedemo.service.AccountService;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/accounts")
+@AllArgsConstructor
+public class AccountController {
+    private final AccountService accountService;
+    private final AccountMapper accountMapper;
+
+    @PostMapping("/add")
```

Comment:
let's review the task description and the URL mappings that we should use for our endpoints

CRUD method names in URLs (like `add` is not a good practice)